service/sandbox-core/src/main/resources/xs2a-application.properties

```diff
@@ -3,7 +3,7 @@ server.port=8080
 spring.application.name=psd2-sandbox-xs2a
 spring.jmx.default-domain=de.adorsys.psd2.xs2a
 # don't serve static files (e.g. index.html from XS2A)
-spring.resources.add-mappings=false
+spring.resources.add-mappings=true
```

Comment:
is that what we need? We don't need xs2a's index.html or?

Reply:
removed this property because it's outdated after splitting certificate generation service